content/1.2/docs/serverless-workflows/examples.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Workflow Examples"
+date: 2024-03-03
+weight: 110
+---
+
+Our Orchestrator Serverless Workflow Examples repository, located at [GitHub](https://github.com/parodos-dev/serverless-workflow-examples), provides a collection of sample workflows designed to help you explore and understand how to build serverless workflows using Orchestrator. These examples showcase a range of use cases, demonstrating how workflows can be developed, tested, and executed based on various inputs and conditions.
```

Comment:
Should the PROD repo be mentioned as well?

Reply:
I'm convinced. since less-is-more, I'll remove the last sentence since I don't find a nice way and wouldn't like to mix prod with example in this context